src/devices/machine/wd_fdc.h

```diff
@@ -99,7 +99,7 @@ class wd_fdc_device_base : public device_t {
 	bool disable_mfm;
 	bool enmf;
 	bool has_enmf;
-	bool inverted_bus;
+	uint8_t bus_invert_value;
```

Comment:
I wasn't really happy with the name of the variable, if someone has a better idea, let me know.